src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml

```diff
@@ -20,138 +19,119 @@
         <div asp-show="Model.HasWarning" class="warning-summary govuk-!-margin-bottom-0">
             <p>@Model.WarningMessage</p>
         </div>
-        <h1 class="govuk-heading-l govuk-!-margin-bottom-6">Your vacancies</h1>
         <div asp-show="Model.HasInfo" class="info-summary govuk-!-margin-bottom-0">
             <p>@Model.InfoMessage</p>
         </div>
     </div>
 </div>
-<div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
 
-        <div asp-hide="Model.HasVacancies">
-            <esfaFeatureEnabled name="@FeatureNames.AllowLevyPayingEmployersOnly">
-                <p class="govuk-body">
-                    You must be a levy-paying employer to post a vacancy.
-                </p>
-                <p class="govuk-body">
-                    You can clone vacancies once they've been submitted.
-                </p>
-                <p class="govuk-body">
-                    Vacancies will be approved or rejected within 24 hours.
-                </p>
-            </esfaFeatureEnabled>
+<partial name="_NoVacanciesContent" model="@Model" />
```

Comment:
If you use asp-show here, then no need to pass the model to the partial view. Also it makes the partial view agnostic.

Reply:
👍 I had assumed this wouldn't work with <partial />. Turns out is does!